articles/java/containers/kubernetes.md

```diff
@@ -23,18 +23,18 @@ Keep in mind that the cost of a VM SKU is proportional to the number of cores an
 
 ## Set CPU requests and limits
 
-If you must limit the CPU, ensure that you apply the same value for both `limits` and `requests` in the deployment file. The JVM doesn't dynamically adjust its runtime, such as the GC and other thread pools. The JVM reads the number of processors available only during startup time.
+For the best fair CPU utilization across your Kubernetes cluster, ensure all containers specify CPU requests, but do not specify CPU limits. If you must limit the CPU, ensure that you apply the same value for both `limits` and `requests` in the deployment file. The JVM doesn't dynamically adjust its runtime, such as the GC and other thread pools. The JVM reads the number of processors available only during startup time.
```

Comment:
We can't commit to this guidance as workloads will have different needs, as well as customers may have different policies and different workloads (of different languages as well).

This guidance, the way it is written here, may be interpreted as a best practice for any Kuberrnetes workload, which in reality it is not true.